application/testing/CMakeLists.txt

```
@@ -1097,6 +1097,10 @@ f3d_test(NAME TestNonExistentConfigFileStem DATA cow.vtp CONFIG "dummy" REGEXP "
 # Test invalid config file
 f3d_test(NAME TestInvalidConfigFile DATA cow.vtp CONFIG ${F3D_SOURCE_DIR}/testing/configs/invalid.json REGEXP "Unable to parse the configuration file" NO_BASELINE)
 
+# Test invalid reset key in config file
+f3d_test(NAME TestInvalidResetKey DATA cow.vtp CONFIG ${F3D_SOURCE_DIR}/testing/configs/invalid_reset.json REGEXP "Invalid option: 'reset-' must be followed by a valid option name, ignoring entry" NO_BASELINE)
```

instead of relying on a config file, it would be better to test with `--reset-` in the command line.

```suggestion
f3d_test(NAME TestInvalidResetOptions DATA cow.vtp ARGS `--reset-` REGEXP "Invalid option: 'reset-' must be followed by a valid option name, ignoring entry" NO_BASELINE)
```

This way you can drop the config file.

---

I reworked the test to use --reset- as suggested, but it doesn’t reach UpdateOptions() — it's rejected earlier by the CLI parser. So either we drop the test or keep it config-based. Let me know what works best for you!